test/check-application

```diff
@@ -1358,6 +1358,8 @@ class TestApplication(testlib.MachineCase):
         b.wait(lambda: self.getContainerAttr(IMG_BUSYBOX, "Memory") != "")
         memory = self.getContainerAttr(IMG_BUSYBOX, "Memory")
         if auth or self.has_cgroupsV2:
+            # Wait for CPU usage to settle before saving CPU usage for comparison later
+            b.wait(lambda: get_cpu_usage(IMG_BUSYBOX) < 25)
```

Comment:
Good catch. Might need a `b.wait_timeout(60)` even, but let's see!

Reply:
Doesn't seem to be needed, I tested this in the test PR and it didn't fail on that so far.

Reply:
:+1: